test/built-ins/Symbol/not-callable.js

```diff
@@ -0,0 +1,25 @@
+// Copyright (C) 2019 Aleksey Shvayka. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+/*---
+esid: sec-properties-of-symbol-instances
+description: >
+  Symbol primitives and objects are not callable.
```

Comment:
to make sure they are really not callable, it might be worth adding a check for `new (Symbol());` (not `new Symbol()`)

Reply:
da71fce